code-challenges.test.js

```diff
@@ -10,45 +10,85 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-var colors1 = ["purple", "blue", "green", "yellow", "pink"]
+var colors1 = ["purple", "blue", "green", "yellow", "pink"];
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+var colors2 = [
+  "chartreuse",
+  "indigo",
+  "periwinkle",
+  "ochre",
+  "aquamarine",
+  "saffron",
+];
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
-
+describe("colorShuffler", () => {
+  const expected1 = ["yellow", "blue", "pink", "green"];
+  const expected2 = ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"];
+  it("returns an array with the first index removed with the rest of the values in the array shuffled", () => {
+    expect(colors1).toEqual(expect.arrayContaining(expected1));
+    expect(colors2).toEqual(expect.arrayContaining(expected2));
+  });
+});
+
```

Comment:
Nice job on the test. Don't forget to bring your test variables down inside the scope of your test.